### PR TITLE
[IMP] inventory: purchase security lead time

### PR DESCRIPTION
**Pull request by Felicious**
## Summary
- Update ref anchors to match current style
- Create "Days to prepare manufacturing order" section
- Create "Days to purchase" section
- Fix Purchase security lead time typo, as shared in this [RD feedback](https://www.odoo.com/web#id=3897481&cids=3&model=project.task&view_type=form)

Merge after Brandon's https://github.com/odoo/documentation/pull/9138/

### CODE REVIEW

**Comment by Felicious** [comment]
Hello @brse-odoo ! It's cool that we're working on separate tickets, but on the same doc! 

To avoid merge conflicts, I'm pointing my PR to yours, until your https://github.com/odoo/documentation/pull/9178 gets merged into 17.0 :blush: 

I saw that you also mentioned the Days to prepare manufacturing order field in your PR. so I moved what you wrote to its own, dedicated section. Feel free to make rewording suggestions however you please, as it was originally your writing :blush: Appreciate your review, whenever you get the chance!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst [hunk 1]
```diff
@@ -16,38 +21,49 @@ process. Here's a summary of the types of lead times in Odoo:
...
   order (SO) is confirmed to the date the products are shipped from the warehouse. This is also
   known as *delivery lead time*.
 
-- :ref:`Sales security lead time <inventory/management/sales-security-lt>`: moves the *scheduled
-  delivery date* forward by a specified number of days. This serves as a buffer to allow the team
-  ample time to prepare the outgoing shipment earlier, considering the possibility of delays in the
-  fulfillment process.
+- :ref:`Sales security lead time <inventory/shipping_receiving/sales-security-lt>`: moves the
+  *scheduled delivery date* forward by a specified number of days. This serves as a buffer to allow
+  the team ample time to prepare the outgoing shipment earlier, considering the possibility of
```
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst:31]:
```suggestion
   ample time to prepare the outgoing shipment earlier, considering the possibility of
```
Optional - "the team" sounded a little off/vague to me

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst [hunk 2]
```diff
@@ -16,38 +21,49 @@ process. Here's a summary of the types of lead times in Odoo:
...
   it takes for products to arrive at the warehouse, facilitating effective scheduling and planning
   of supplier deliveries.
 
-- :ref:`Purchase security lead time <inventory/management/purchase-security-lt>`: advances the order
-  deadline on a :abbr:`PO (Purchase Order)` by a specified number of days. This proactive approach
-  of placing orders earlier mitigates the risk of vendor or shipping delays. Thus, for products that
-  are set to replenish to order, the need appears on the *Replenishment report* earlier, according
-  to the specified number of days.
+- :ref:`Purchase security lead time <inventory/shipping_receiving/purchase-security-lt>`: advances
+  the order deadline on a :abbr:`PO (Purchase Order)` by a specified number of days. This proactive
```
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst:40]: If you want you can also do a replace at the top for PO, like you did the other abbr tags

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst [hunk 3]
```diff
@@ -246,19 +276,24 @@ manufacturing lead times and manufacturing security lead times.
 Manufacturing lead time
 -----------------------
 
-Configure the manufacturing lead time directly on the product form, by navigating to
-:menuselection:`Manufacturing app --> Products --> Products`, and select the desired product.
+Manufacturing lead times for products are configured from a product's Bill of Materials (BoM) form.
```
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst:279]:
```suggestion
Manufacturing lead times for products are configured from a product's bill of materials (BoM) form.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst [hunk 4]
```diff
@@ -269,28 +304,54 @@ manufacturing lead time.
...
+Days to prepare manufacturing order
+-----------------------------------
+
+Configure the days required to gather components to manufacture a product by going to its |BoM|. To
+do that, go to :menuselection:`Manufacturing app --> Products --> Bills of Materials`, and select
+the desired |BoM|.
+
+In the :guilabel:`Miscellaneous` tab of the |BoM|, specify the calendar days needed to obtain
+components of the product in the :guilabel:`Days to prepare Manufacturing Order` field. Doing so
+creates |MOs| in advance, ensure there is enough time to replenish components, or manufacture
```
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst:332]:
```suggestion
creates |MOs| in advance, and ensures there is enough time to either replenish components, or manufacture
```

**Review by brse-odoo — APPROVED** [review]
@Felicious Just finished reviewing this - great job. My PR is merging as I write this so your should be good to go soon. Let me know if you have any questions

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst [hunk 5]
```diff
@@ -16,38 +21,49 @@ process. Here's a summary of the types of lead times in Odoo:
...
+  products that are set to replenish to order, the need appears on the *Replenishment report*
+  earlier, according to the specified number of days.
+
+- :ref:`Days to Purchase <inventory/shipping_receiving/days-to-purchase>`: days needed for the
+  vendor to receive a request for quotation (RFQ) and confirm it. It advances the deadline to
+  schedule a |RFQ| by a specified number of days.
 
-- :ref:`Manufacturing lead time <inventory/management/manuf-lt>`: the number of days needed to
-  complete a manufacturing order (MO) from the date of confirmation. This lead time includes
+- :ref:`Manufacturing lead time <inventory/shipping_receiving/manuf-lt>`: the number of days needed
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst:49]:
```suggestion
- :ref:`Manufacturing lead time <inventory/shipping_receiving/manuf-lt>`: number of days needed
```

**Review by ksc-odoo — APPROVED** [review]
Hey @Felicious -- just finished my Final Review. Approving now. Once you address the comments I left, and make all the necessary changes, you can feel free to tag this for Tech Review. Thanks! 👍

**Comment by Felicious** [comment]
Hello @samueljlieber ! This is ready for your tech review (:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates/manuf-security.png
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates/manuf-security.png:1]: There isnt much of a change at all between these two images, I dont think the new one is worth the 310% jump in diff :)

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious, thanks for your work on this PR! Your changes look good to me, approving and delegating with one comment. Thank you!
..
@robodoo delegate=Felicious

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates/manuf-security.png
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates/manuf-security.png:1]: HAHA ok! I see your point. I'll crop the image.

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**